src/utils/__tests__/message-test.js

```diff
@@ -196,3 +206,51 @@ describe('findFirstUnread', () => {
     expect(result).toEqual(unreadMessage);
   });
 });
+
+describe('getLinkToMessage', () => {
+  test('should return a stream link if message link copied from stream', () => {
+    const auth = { realm: 'https://www.foo.bar/' };
```

Comment:
In other place, we are using just using `zulip.com` or `example`. I would not prefer to have one more category like `goo.bar` :)

Reply:
I will fix this, thanks! 🙂